src/group_theory/perm/cycle_type.lean

```diff
@@ -5,6 +5,7 @@ Authors: Thomas Browning
 -/
 
 import combinatorics.partition
+import ring_theory.int.basic
 import algebra.gcd_monoid.multiset
 import tactic.linarith
 import group_theory.perm.cycles
```

Comment:
Let's take the opportunity to sort the imports here.
```suggestion
import algebra.gcd_monoid.multiset
import combinatorics.partition
import group_theory.perm.cycles
import group_theory.sylow
import ring_theory.int.basic
import tactic.linarith
```

I can't get rid of the `import group_theory.sylow` line below, since GitHub doesn't let me comment on it. Please delete it if you apply this change!